app/fruits/page.tsx

```diff
@@ -10,34 +10,29 @@ export const metadata = {
 
 export default function FruitsPage() {
   // get the cookie from the server
-  const fruitsCookie = cookies().get('fruitsCookie');
+  const fruitComments = parseJson(getCookie('fruitComments'));
 
-  // create a default value if cooke doesn't exist
-  let fruitsCookieParsed: CookieValue = [];
+  const currentComments = Array.isArray(fruitComments) ? fruitComments : [];
```

Comment:
- `currentComments` seems like a confusing name here, maybe it can be `fruitComments` instead
- in order to use `fruitComments`, the line 13 above needs to change (it will need to be split into 2 lines anyway, so that parsing happens on a different line than getting the cookie)

Reply:
1226390